eisbuk-admin/src/components/atoms/IntervalCard/IntervalCard.tsx

```diff
@@ -64,30 +65,13 @@ const BookingCard: React.FC<Props> = ({
    * - you need to refactor this to use interval instead of calculating from duration
    * - should be straight forward, no fancy calculations or conversions (the start and end times are already received as `interval` properties as strings)
    */
```

Comment:
This part is resolved, so you can remove this comment (`@FOR_FADWA`) as well as the below one (`TASK_2`)